### PR TITLE
Feature/04 2024 complete add employee

### PR DESCRIPTION
**Pull request by solomoreal**
## Complete add employees
- handle CSV upload and processing
- handle manual add employee
- send a welcome email to the employee
- update the email template
- remove tailwind config and style from the email template to app.css and tailwing_config respectively
- make select input configurable
- clear manual input after submit

### CODE REVIEW

File: resources/views/livewire/create-employers.blade.php [hunk 1]
```diff
@@ -194,14 +214,14 @@ class="font-semibold text-[12px] text-[#4F4F4F]"
 
                             <select
                               id="pay"
-                              class="h-[50px] w-full flex items-center rounded-[8px] border-[1px] border-[#E6E6E6] bg-white-0 text-[12px] font-medium placeholder:text-[#A7A7A7] px-[24px] @error("employees.{{$i}}.pay_rate") border-red-500 @enderror"
-                              wire:model="employees.{{$i}}.pay_rate"
+                              class="h-[50px] w-full flex items-center rounded-[8px] border-[1px] border-[#E6E6E6] bg-white-0 text-[12px] font-medium placeholder:text-[#A7A7A7] px-[24px] @error("employees.$i.pay_rate") border-red-500 @enderror"
+                              wire:model.live="employees.{{$i}}.pay_rate"
                             >
                               <option selected>Enter range</option>
```
Comment (worldwayii) [inline resources/views/livewire/create-employers.blade.php:220]: Can we please make this configurable? also `Employee Position/ Job Role` as well? Bearing in mind this job role can grow 6 to 50+ tomorrow?

File: resources/views/emails/employee/created.blade.php [hunk 2]
```diff
@@ -1,26 +1,209 @@
...
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/flowbite/2.2.1/flowbite.min.css"
+      rel="stylesheet"
+    />
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script>
+      tailwind.config = {
```
Comment (worldwayii) [inline resources/views/emails/employee/created.blade.php:18]: is there a need for this? we already have a tailwind.config in the project

File: resources/views/emails/employee/created.blade.php [hunk 3]
```diff
@@ -1,26 +1,209 @@
...
+            },
+          },
+          fontFamily: {
+            montserrat: ["Montserrat", "sans-serif"],
+          },
+        },
+      };
+    </script>
+    <title>Welcome To Kcontrack</title>
+    <style>
```
Comment (worldwayii) [inline resources/views/emails/employee/created.blade.php:75]: can this style belong to the app.css and pulled in here? so it it re-useable?

**Review by worldwayii — CHANGES_REQUESTED** [review]
Also Can we address the changes  mentioned on whatsapp? When creating and employee with form and it fails, errors don get flashed.

Nice work overall sir

**Comment by solomoreal** [comment]
> Also Can we address the changes mentioned on whatsapp? When creating and employee with form and it fails, errors don get flashed.
> 
> Nice work overall sir

handled

File: resources/views/emails/employee/created.blade.php (hunk 3 above)
Comment (solomoreal) [reply resources/views/emails/employee/created.blade.php]: > can this style belong to the app.css and pulled in here? so it it re-useable?

handled

File: resources/views/livewire/create-employers.blade.php (hunk 1 above)
Comment (solomoreal) [reply resources/views/livewire/create-employers.blade.php]: done

File: resources/views/emails/employee/created.blade.php (hunk 2 above)
Comment (solomoreal) [reply resources/views/emails/employee/created.blade.php]: done

File: resources/views/emails/employee/created.blade.php (hunk 3 above)
Comment (solomoreal) [reply resources/views/emails/employee/created.blade.php]: done

**Pull request merged**